test/cljam/t_dict.clj

```diff
@@ -2,9 +2,26 @@
   "Tests for cljam.dict."
   (:require [clojure.test :refer :all]
             [me.raynes.fs :as fs]
+            [clojure.java.io :as io]
             [cljam.t-common :refer :all]
             [cljam.dict :as dict]))
 
+(defn same-dict-file? [f1 f2]
+  (with-open [r1 (io/reader f1)
+              r2 (io/reader f2)]
+    (let [dict1 (line-seq r1)
+          dict2 (line-seq r2)
+          ;; NB: `UR` is calculated by file path, ignore it
+          omit-ur (fn [line]
+                    (if-let [[_ r] (re-find #"^(.*)\tUR:" line)]
+                      r
+                      line))]
+      (when (= (count dict1) (count dict2))
+        (->> (map #(= (omit-ur %1) (omit-ur %2)) dict1 dict2)
+             (filter identity)
+             first
+             boolean)))))
```

Comment:
Is this implementation right? This seems to return `false` only if all lines are different. (`true` if any pair of lines is equal.)

Is the following enough?

```clojure
(->> (map #(= (omit-ur %1) (omit-ur %2)) dict1 dict2)
     (every? true?))
```

Additionally, `VN:1.4` can be ignore. `omit-vn` is required.